riff-raff/app/conf/context.scala

```diff
@@ -219,6 +219,26 @@ class Config(configuration: TypesafeConfig, startTime: DateTime) extends Logging
     }
   }
 
+  object management {
+    object aws {
+      // TODO read config from more generic location
+      lazy val regionName: String = getStringOpt("scheduledDeployment.aws.region").getOrElse("eu-west-1")
+
+      // For compatibility reasons this need to use the old AWS SDK
+      lazy val snsClient: AmazonSNSAsync = AmazonSNSAsyncClientBuilderV1.standard()
+        .withCredentials(credentialsProviderChainV1(None, None))
+        .withRegion(regionName)
+        .build()
+
+      // TODO read config from more generic location
+      lazy val anghammaradTopicARN: String = getString("scheduledDeployment.anghammaradTopicARN")
+
+      lazy val ec2Client: Ec2Client = Ec2Client.builder()
+        .credentialsProvider(credentialsProviderChain(None, None))
```

Comment:
I notice you're not using the v1 chain - is that correct?

Reply:
Minor, but shall we leave off the `(None, None)` since those are the defaults anyway

Reply:
The v2 client is used here, on the assumption that's the direction of travel we're going in and v1 should only be used for compat reasons. `credentialsProviderChain` does not default it's arguments, so `None, None` is still needed.

Would suggest we (separately) refactor to improve naming - `credentialsProviderChainV1` becomes `legacyCredentialsProviderChain`, for example.

Reply:
https://github.com/guardian/riff-raff/pull/696